### ISSUE TITLE
Time entries

### ISSUE THREAD

**Issue by rNoz** [alphanodes/redmine_messenger#45]
Any possibility to support time entries? I know that this can be quite verbose, but for specific projects/small teams can be interesting.

In case you do not plan to support it, do you recommend me to proceed like this: starting with issue_patch.rb, but adapting it for time entries, along with a boolean option to enable/disable it in the settings?

**Comment by ZerooCool** [comment]
Same ?
https://github.com/AlphaNodes/redmine_messenger/issues/50

**Comment by rNoz** [comment]
Not exactly. How do you achieve time entries being logged in Rocket? I can only see issue modifications, but not time entries. Maybe I missed some config?

**Comment by ZerooCool** [comment]
My tests were only on Slack.
Issue # 50 has been taken into account and resolved for Slack.

For Rocket it will be taken into consideration.
